kimchi/src/circuits/polynomials/chacha.rs

```diff
@@ -554,7 +554,7 @@ mod tests {
                 h.insert(Column::Witness(i));
             }
             for i in 0..(lookup_info.max_per_row + 1) {
-                h.insert(Column::LookupSorted(i));
+                h.insert(Column::LookupSorted(i as usize));
```

Comment:
Nit: `LookupSorted` should also become `u32` (although not necessarily in this PR)

Reply:
I tried that but this led to another rabbit hole so made a minimal PR